README.md

```diff
@@ -130,97 +133,98 @@ user = "tegola"             # postgis database user (required)
+
+  [[providers.layers]]
+  name = "roads"                      # will be encoded as the layer name in the tile
+  tablename = "gis.zoning_base_3857"  # sql or tablename are required
+  geometry_fieldname = "geom"         # geom field. default is geom
+  geometry_type = "linestring"        # geometry type. if not set, tables are inspected at startup to try and infer the gemetry type
+  id_fieldname = "gid"                # geom id field. default is gid
+  fields = [ "class", "name" ]        # Additional fields to include in the select statement.
+
+  [[providers.layers]]
+  name = "rivers"                     # will be encoded as the layer name in the tile
+  geometry_fieldname = "geom"         # geom field. default is geom
+  id_fieldname = "gid"                # geom id field. default is gid
+  # Custom sql to be used for this layer. Note: that the geometery field is wraped
+  # in a ST_AsBinary() and the use of the !BBOX! token
+  sql = "SELECT gid, ST_AsBinary(geom) AS geom FROM gis.rivers WHERE geom && !BBOX!"
+
+  [[providers.layers]]
+  name = "buildings"                  # will be encoded as the layer name in the tile
+  geometry_fieldname = "geom"         # geom field. default is geom
+  id_fieldname = "gid"                # geom id field. default is gid
+  # Custom sql to be used for this layer as a sub query. ST_AsBinary and
+  # !BBOX! filter are applied automatically.
+  sql = "(SELECT gid, geom, type FROM buildings WHERE scalerank = !ZOOM! LIMIT 1000) AS sub"
 
 # maps are made up of layers
 [[maps]]
-name = "zoning"                              # used in the URL to reference this map (/maps/:map_name)
+name = "zoning"                              # used in the URL to reference this map (/maps/zoning)
 
-	[[maps.layers]]
-	name = "landuse"                         # name is optional. If it's not defined the name of the ProviderLayer will be used.
+  [[maps.layers]]
+  name = "landuse"                         # name is optional. If it's not defined the name of the ProviderLayer will be used.
 	                                         # It can also be used to group multiple ProviderLayers under the same namespace.
-	provider_layer = "test_postgis.landuse"  # must match a data provider layer
-	min_zoom = 12                            # minimum zoom level to include this layer
-	max_zoom = 16                            # maximum zoom level to include this layer
+  provider_layer = "test_postgis.landuse"  # must match a data provider layer
+  min_zoom = 12                            # minimum zoom level to include this layer
+  max_zoom = 16                            # maximum zoom level to include this layer
 
-		[maps.layers.default_tags]           # table of default tags to encode in the tile. SQL statements will override
-		class = "park"
+    [maps.layers.default_tags]           # table of default tags to encode in the tile. SQL statements will override
+    class = "park"
 
 	[[maps.layers]]
```

Comment:
Why was the spacing of this not changed to line up with the  `[[maps.layers]]` before it.